index.html

```diff
@@ -2517,26 +2518,68 @@ <h3>
         <dl>
           <dt>@context</dt>
           <dd>
-The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. All members of the
+The value of the <code>@context</code> property MUST be exactly one of the following:
+
+<ul>
+  <li>The string <code>https://www.w3.org/ns/did/v1</code>. 
+    <pre class="example">
+      {
+        "@context": "https://www.w3.org/ns/did/v1",
+        "id": "did:example:123"
+      }      
+    </pre>
+  </li>
+  <li>An array of one or more elements,
+    where the value types of the members of the array
+    MUST be either a string or an object and the first element is
+    the string <code>https://www.w3.org/ns/did/v1</code>.
```

Comment:
Try to use Infra types for "array", "string", and "object".

Reply:
done in https://github.com/w3c/did-core/pull/408/commits/f9f1ee80144cc434533b530c23c66c83c33bcda9